leases/app/lib/LeaseNotifier.scala

```diff
@@ -2,10 +2,36 @@ package lib
 
 import com.gu.mediaservice.lib.aws.{MessageSender, UpdateMessage}
 import com.gu.mediaservice.lib.formatting._
-import com.gu.mediaservice.model.{LeaseNotice, LeasesByMedia, MediaLease}
+import com.gu.mediaservice.model.{LeasesByMedia, MediaLease}
+
 import org.joda.time.DateTime
 import play.api.libs.json._
 
+case class LeaseNotice(mediaId: String, leaseByMedia: JsValue) {
```

Comment:
I think this is a bad rebase as this case class has been moved to `common-lib`?

Reply:
Good catch!